### PR TITLE
src: Increase the asshat 11111 limit to 99999

### PR DESCRIPTION
**Pull request by hmv47**

### CODE REVIEW

File: Rule34.xxx Downloader/DownloadingForm.cs [hunk 1]
```diff
@@ -16,14 +16,14 @@ public DownloadingForm(int maxPid)
 
         private void DownloadingForm_Load(object sender, EventArgs e)
         {
-            if (maxPid < 11111)
+            if (maxPid < 99999)
```
Comment (jokset) [inline Rule34.xxx Downloader/DownloadingForm.cs:19]: extract to a constant.

**Review by jokset — APPROVED** [review]
where is jira ticket.

**Comment by hmv47** [comment]
> where is jira ticket.

Jira was shown to induce lung cancer in mice as early as 1943

**Comment by jokset** [comment]
> 

it's been proven in 2007 by atlassian developers that those mice were fed by software engineers thus the cancer

**Pull request merged**